src/pybind/mgr/dashboard/ci/cephadm/ceph_cluster.yml

```diff
@@ -8,7 +8,7 @@ parameters:
  prefix: ceph
  numcpus: 1
  memory: 2048
- image: fedora36
+ image: /var/lib/libvirt/images/ceph-dashboard/fedora36
```

Comment:
do we really need an abs path here? isn't kcli able to find an image by name?

Reply:
just a typo, removed